frontend/src/app/models/user-rail-profile.ts

```diff
@@ -0,0 +1,4 @@
+export interface UserRailProfile {
```

Comment:
I wonder whether we should put the definition of this interface into `.../models/backend-generated` folder and then in the `...models/` folder we create a new interface that inherits from the previous one. At least this is an approach we've adhered to so far.

Reply:
In such a case, we might want to also create two definitions of classes corresponding to `RailUserCreate` and `RailUserResponse` respectively

Reply:
I agree, that's the proper way.

I'd suggest doing so in next iteration, though. It's quite possible it's gonna change and the resources are low now.